web/src/beta/lib/core/engines/Cesium/Feature/index.tsx

```diff
@@ -26,7 +26,16 @@ export * from "./utils";
 export { context, type Context } from "./context";
 export { getTag } from "./utils";
 
-const components: Record<keyof AppearanceTypes, [FeatureComponent, FeatureComponentConfig]> = {
+const NON_RENDERABLE_APPEARANCE = ["transition"] as const;
+const isRenderableAppearance = (
+  k: keyof AppearanceTypes,
+): k is Exclude<keyof AppearanceTypes, (typeof NON_RENDERABLE_APPEARANCE)[number]> =>
+  !(NON_RENDERABLE_APPEARANCE as unknown as string[]).includes(k);
```

Comment:
The casting (NON_RENDERABLE_APPEARANCE as unknown as string[]) is a bit awkward. It's necessary due to the as const assertion, but it might be clearer to define NON_RENDERABLE_APPEARANCE without as const if the tuple does not need to be immutable.

Reply:
I want to keep this being type safe, so I fixed it like this. How do you think?:pray:
https://github.com/reearth/reearth/pull/860/commits/dd1c428b46ac7b6dcb1c9707369b3f37f981f591

Reply:
Looks Neat, Thanks.